src/app/core/services/misc/localization.service.ts

```diff
@@ -59,6 +60,14 @@ export class LocalizationService {
     return this.language
   }
 
+  updateLanguageSettings(): Promise<any> {
```

Comment:
is this required for when migrating to a new version of the app? In that case, this https://github.com/RADAR-base/RADAR-Questionnaire/issues/264 will be directly useful.

Reply:
Yeah so that it would update when new languages are added to a new version, or if languages are added to the remote config (in the future?).